common/djangoapps/student/views.py

```diff
@@ -211,9 +211,20 @@ def index(request, extra_context=None, user=AnonymousUser()):
 
     context['programs_list'] = get_programs_with_type(include_hidden=False)
 
+    if third_party_auth.is_enabled() and not request.user.is_authenticated():
+        context['tpa_hint'] = get_auth0_provider_id()
+
     return render_to_response('index.html', context)
 
 
+def get_auth0_provider_id():
+    """Get Auth0 provider id if enabled."""
```

Comment:
edX Code-style suggests to refrain from using single-lined doc-strings.
```suggestion
    """
    Get Auth0 provider id if enabled.
    """
```